src/modules/bot/bot.service.spec.ts

```diff
@@ -735,4 +735,13 @@ describe('BotService', () => {
     deletedIds = [];
     fetchMock.restore();
   });
+  it('should return bot IDs in the response', async () => {
+    fetchMock.delete(`${configService.get<string>('UCI_CORE_BASE_URL')}${configService.get<string>('CAFFINE_INVALIDATE_ENDPOINT')}`,
+    true
+    );
+    const response = await botService.remove({ids: ['testId1', 'testId2'], endDate: null});
+    // expect(Array.isArray(response)).toBe(true);
+    const expectedBotIds = ['testId1', 'testId2'];
+    expect(response).toEqual(expectedBotIds);
```

Comment:
This will not work since the mockBotDb does not contain any ids corresponding to `testId1` or `testId2`, hence this will not be added in the final result.